xml/sip_user_agents.xml

```diff
@@ -129,6 +129,36 @@
     <param pos="0" name="os.cpe23" value="cpe:/o:cisco:ios:{os.version}"/>
   </fingerprint>
 
+  <fingerprint pattern="^Cisco-CP(\d{4})/([\d\.]+\d)$">
+    <description>Cisco CP-model IP Phones</description>
+    <example cisco.model="3905" hw.product="CP-3905" os.version="9.4.1">Cisco-CP3905/9.4.1</example>
+    <param pos="1" name="cisco.model"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.product" value="CP-{cisco.model}"/>
```

Comment:
IMO, this should be `Unified SIP Phone {cisco.model} 


```
h:cisco:unified_sip_phone_3911
h:cisco:unified_sip_phone_3951
```

The Cisco 3900 series phones are unified SIP phones. The 7xxx series are not necessarily.

There is another fingerprint in the area that needs to be reworked as well.  If folks don't mind we can create a follow  up PR to sort this out and have folks comment on it. 

Reference: https://www.cisco.com/c/en/us/support/collaboration-endpoints/index.html